cwfloppy.h

```diff
@@ -60,6 +60,11 @@ int catweasel_read(catweasel_drive *d, int side, int clock, int time, int idx);
 
 /* Write data -- msdelay will be used */
 /* If time = 0, write from index hole to index hole */
+/* Return values:
```

Comment:
Wish the function's descriptive comments were with the definition, not the declaration.  Once a declaration is placed in a header file, I would expect few devs would look in a header file for a function's comment  block about what the function does and/or returns.